### PR TITLE
Get and set hold for MHK2

### PR DESCRIPTION
**Pull request by nickwynja**
@dlarrick Thanks for this great tool! The one thing I found missing was getting and setting holds. Especially setting a holiday hold when I'm on vacation and forgot to set it from home.

I have an MHK2 so am not able to test this in other scenarios without that thermostat so maybe someone else can help test in other conditions.

### CODE REVIEW

**Comment by dlarrick** [comment]
Thanks for your contribution.

I think most users are using pykumo with something like Home Assistant so there hasn't been the need to use holds on the equipment itself.

That said, I can't see any harm in adding these new functions since they're entirely new. If you could please bump the version (in setup.py) to 0.3.0 in your PR that would make my life a little easier.

FYI the indoor units (at least mine) have the capability to do schedules on their own. I haven't played with that at all (see: Home Assistant) but for users with spotty network connections it might be more reliable than counting on being able to send a command at the right time. If you're interested they're under `{"c":{"indoorUnit":{"schedule":{}}}}`

File: pykumo/py_kumo.py [hunk 1]
```diff
@@ -262,6 +263,32 @@ def get_defrost(self):
...
         return val
 
+    def get_hold_time(self):
+        """ Get hold time from MHK2 """
+        query = '{"c":{"mhk2":{"hold":{"adapter":{"endTime":{}}}}}}'.encode('utf-8')
+        response = self._request(query)
+        try:
+            end_time = response['r']['mhk2']['hold']['adapter']['endTime']
+        except KeyError:
+            end_time = None
```
Comment (cjkrolak) [inline pykumo/py_kumo.py:273]: end_time = None is going to trigger a TypeError down at line 284.  Maybe defaulting to 0 is a better choice?

Comment (nickwynja) [reply pykumo/py_kumo.py:273]: @cjkrolak Thanks! Pushed a change where I log in this case and set to "".

**Comment by nickwynja** [comment]
I've bumped the version and addressed the `TypeError`.

> FYI the indoor units (at least mine) have the capability to do schedules on their own. I haven't played with that at all (see: Home Assistant) but for users with spotty network connections it might be more reliable than counting on being able to send a command at the right time. If you're interested they're under `{"c":{"indoorUnit":{"schedule":{}}}}`

I'll have to take another look at Home Assistant. I do use the schedules directly on the MHK2 and they generally work the way I need them too. This addition will help to use pykumo with a script to temporarily override the schedule similar to how it works in the (not so great) kumo app.

**Comment by dlarrick** [comment]
Just to set expectations, I plan to look more closely this weekend and likely merge & make a new release then.

FYI my comment about Home Assistant was not intended to imply it can use the units' scheduling; but rather, that it enables much more nuanced control than any standalone device could offer -- if you're willing to put the work into the automations.

**Review by dlarrick — APPROVED** [review]

**Pull request merged**